Dailybuild Rollback Failure - entity content is too long [377716148] for the configured buffer limit [104857600].

Our QA server runs into this error when it tries to do a scheduled daily build import. SNOMEDCT-NO gets locked because of this.

Full stack trace:

```
Dec 14 14:19:12 hostname java[24565]: 2021-12-14 14:19:12.222 DEBUG 24565 --- [   scheduling-1] o.s.s.dailybuild.DailyBuildService       : Daily build resources path 'file:NO/SNOMEDCT-NO'.
Dec 14 14:19:12 hostname java[24565]: 2021-12-14 14:19:12.224 DEBUG 24565 --- [   scheduling-1] o.s.s.dailybuild.DailyBuildService       : Found total builds 68 from file:NO/SNOMEDCT-NO
Dec 14 14:19:12 hostname java[24565]: 2021-12-14 14:19:12.224  INFO 24565 --- [   scheduling-1] o.s.s.dailybuild.DailyBuildService       : New daily build 2021-12-14-125652.zip found for SNOMEDCT-NO
Dec 14 14:19:12 hostname java[24565]: 2021-12-14 14:19:12.461  INFO 24565 --- [   scheduling-1] o.s.s.dailybuild.DailyBuildService       : Latest version found at 1634542601397 on MAIN/SNOMEDCT-NO.
Dec 14 14:19:20 hostname java[24565]: 2021-12-14 14:19:20.451 ERROR 24565 --- [   scheduling-1] o.s.s.d.ScheduledDailyBuildImportService : Failed to import daily build for code system SNOMEDCT-NO
Dec 14 14:19:20 hostname java[24565]: org.springframework.dao.DataAccessResourceFailureException: entity content is too long [377716148] for the configured buffer limit [104857600]; nested exception is java.lang.RuntimeException: entity content is too long [37771
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.ElasticsearchExceptionTranslator.translateExceptionIfPossible(ElasticsearchExceptionTranslator.java:76)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.translateException(ElasticsearchRestTemplate.java:398)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.execute(ElasticsearchRestTemplate.java:381)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.search(ElasticsearchRestTemplate.java:289)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.AbstractElasticsearchTemplate.search(AbstractElasticsearchTemplate.java:331)
Dec 14 14:19:20 hostname java[24565]: at org.snomed.snowstorm.core.data.services.SBranchService.findAllVersionsAfterOrEqualToTimestamp(SBranchService.java:99)
Dec 14 14:19:20 hostname java[24565]: at org.snomed.snowstorm.dailybuild.DailyBuildService.rollbackDailyBuildContent(DailyBuildService.java:128)
Dec 14 14:19:20 hostname java[24565]: at org.snomed.snowstorm.dailybuild.DailyBuildService.dailyBuildDeltaImport(DailyBuildService.java:96)
Dec 14 14:19:20 hostname java[24565]: at org.snomed.snowstorm.dailybuild.DailyBuildService.performScheduledImport(DailyBuildService.java:85)
Dec 14 14:19:20 hostname java[24565]: at jdk.internal.reflect.GeneratedMethodAccessor150.invoke(Unknown Source)
Dec 14 14:19:20 hostname java[24565]: at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.lang.reflect.Method.invoke(Method.java:566)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:344)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:198)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:774)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691)
Dec 14 14:19:20 hostname java[24565]: at org.snomed.snowstorm.dailybuild.DailyBuildService$$EnhancerBySpringCGLIB$$f1815633.performScheduledImport(<generated>)
Dec 14 14:19:20 hostname java[24565]: at org.snomed.snowstorm.dailybuild.ScheduledDailyBuildImportService.scheduledDailyBuildDeltaImport(ScheduledDailyBuildImportService.java:63)
Dec 14 14:19:20 hostname java[24565]: at jdk.internal.reflect.GeneratedMethodAccessor149.invoke(Unknown Source)
Dec 14 14:19:20 hostname java[24565]: at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.lang.reflect.Method.invoke(Method.java:566)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:84)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
Dec 14 14:19:20 hostname java[24565]: at java.base/java.lang.Thread.run(Thread.java:829)
Dec 14 14:19:20 hostname java[24565]: Caused by: java.lang.RuntimeException: entity content is too long [377716148] for the configured buffer limit [104857600]
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.translateException(ElasticsearchRestTemplate.java:396)
Dec 14 14:19:20 hostname java[24565]: ... 29 common frames omitted
Dec 14 14:19:20 hostname java[24565]: Caused by: java.io.IOException: entity content is too long [377716148] for the configured buffer limit [104857600]
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:828)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.performRequest(RestClient.java:248)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.performRequest(RestClient.java:251)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.performRequest(RestClient.java:251)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.performRequest(RestClient.java:235)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1609)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1579)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1549)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestHighLevelClient.search(RestHighLevelClient.java:1065)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.lambda$search$11(ElasticsearchRestTemplate.java:289)
Dec 14 14:19:20 hostname java[24565]: at org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate.execute(ElasticsearchRestTemplate.java:379)
Dec 14 14:19:20 hostname java[24565]: ... 28 common frames omitted
Dec 14 14:19:20 hostname java[24565]: Suppressed: java.io.IOException: entity content is too long [377716148] for the configured buffer limit [104857600]
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:828)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.performRequest(RestClient.java:248)
Dec 14 14:19:20 hostname java[24565]: ... 36 common frames omitted
Dec 14 14:19:20 hostname java[24565]: Suppressed: java.io.IOException: entity content is too long [377716148] for the configured buffer limit [104857600]
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:828)
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.RestClient.performRequest(RestClient.java:248)
Dec 14 14:19:20 hostname java[24565]: ... 35 common frames omitted
Dec 14 14:19:20 hostname java[24565]: Caused by: org.apache.http.ContentTooLongException: entity content is too long [377716148] for the configured buffer limit [104857600]
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.HeapBufferedAsyncResponseConsumer.onEntityEnclosed(HeapBufferedAsyncResponseConsumer.java:76)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.nio.protocol.AbstractAsyncResponseConsumer.responseReceived(AbstractAsyncResponseConsumer.java:137)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.MainClientExec.responseReceived(MainClientExec.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseReceived(DefaultClientExchangeHandlerImpl.java:151)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.responseReceived(HttpAsyncRequestExecutor.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:255)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:114)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591)
Dec 14 14:19:20 hostname java[24565]: ... 1 common frames omitted
Dec 14 14:19:20 hostname java[24565]: Caused by: org.apache.http.ContentTooLongException: entity content is too long [377716148] for the configured buffer limit [104857600]
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.HeapBufferedAsyncResponseConsumer.onEntityEnclosed(HeapBufferedAsyncResponseConsumer.java:76)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.nio.protocol.AbstractAsyncResponseConsumer.responseReceived(AbstractAsyncResponseConsumer.java:137)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.MainClientExec.responseReceived(MainClientExec.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseReceived(DefaultClientExchangeHandlerImpl.java:151)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.responseReceived(HttpAsyncRequestExecutor.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:255)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:114)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591)
Dec 14 14:19:20 hostname java[24565]: ... 1 common frames omitted
Dec 14 14:19:20 hostname java[24565]: Caused by: org.apache.http.ContentTooLongException: entity content is too long [377716148] for the configured buffer limit [104857600]
Dec 14 14:19:20 hostname java[24565]: at org.elasticsearch.client.HeapBufferedAsyncResponseConsumer.onEntityEnclosed(HeapBufferedAsyncResponseConsumer.java:76)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.nio.protocol.AbstractAsyncResponseConsumer.responseReceived(AbstractAsyncResponseConsumer.java:137)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.MainClientExec.responseReceived(MainClientExec.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseReceived(DefaultClientExchangeHandlerImpl.java:151)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.responseReceived(HttpAsyncRequestExecutor.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:255)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:114)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
Dec 14 14:19:20 hostname java[24565]: at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591)
Dec 14 14:19:20 hostname java[24565]: ... 1 common frames omitted
Dec 14 14:19:20 hostname java[24565]: 2021-12-14 14:19:20.452 DEBUG 24565 --- [   scheduling-1] o.s.s.c.d.s.i.IdentifierService          : SCTID Registration process disabled.
```

--- a/src/main/java/org/snomed/snowstorm/core/data/services/SBranchService.java
+++ b/src/main/java/org/snomed/snowstorm/core/data/services/SBranchService.java
@@ -89,13 +89,15 @@ public class SBranchService {
 		return branch;
 	}
 
-	public Page<Branch> findAllVersionsAfterOrEqualToTimestamp(String path, Date timestamp, Pageable pageable) {
+	public Page<Branch> findAllVersionsAfterOrEqualToTimestampAsLightCommits(String path, Date timestamp, Pageable pageable) {
 		NativeSearchQueryBuilder queryBuilder = new NativeSearchQueryBuilder()
 				.withQuery(boolQuery()
 						.must(QueryBuilders.termQuery("path", path))
 						.must(QueryBuilders.rangeQuery("start").gte(timestamp.getTime())))
+				.withFields("path", "start", "end", "head", "base", "locked")
 				.withSort(SortBuilders.fieldSort("start"))
 				.withPageable(pageable);
+
 		SearchHits<Branch> searchHits = elasticsearchTemplate.search(queryBuilder.build(), Branch.class);
 		return new PageImpl<>(searchHits.get().map(SearchHit::getContent).collect(Collectors.toList()),
 				pageable, searchHits.getTotalHits());

--- a/src/main/java/org/snomed/snowstorm/dailybuild/DailyBuildService.java
+++ b/src/main/java/org/snomed/snowstorm/dailybuild/DailyBuildService.java
@@ -125,36 +125,37 @@ public class DailyBuildService {
 		Date baseForReleaseCommit = null;
 		Date baseForUpgradeCommit = null;
 		if (releaseCommitHead != null) {
-			List<Branch> commits = sBranchService.findAllVersionsAfterOrEqualToTimestamp(branchPath, releaseCommitHead, Pageable.unpaged()).getContent();
-			logger.info("{} commits found on {} since latest release.", commits.size() - 1, branchPath);
-			for (Branch commit : commits) {
+			List<Branch> lightCommits = sBranchService.findAllVersionsAfterOrEqualToTimestampAsLightCommits(branchPath, releaseCommitHead, Pageable.unpaged()).getContent();
+			logger.info("{} commits found on {} since latest release.", lightCommits.size() - 1, branchPath);
+			for (Branch lightCommit : lightCommits) {
 				// Don't rollback the release commit
-				if (commit.getHead().equals(releaseCommitHead)) {
-					baseForReleaseCommit = commit.getBase();
+				if (lightCommit.getHead().equals(releaseCommitHead)) {
+					baseForReleaseCommit = lightCommit.getBase();
 					commitsToRollback.clear();
 					logger.info("Release commit found, base version {} recorded.", baseForReleaseCommit.getTime());
 					continue;
 				}
 				// Exclude additional upgrade commits after code system version
-				if (baseForUpgradeCommit != null && !commit.getBase().equals(baseForUpgradeCommit)) {
+				if (baseForUpgradeCommit != null && !lightCommit.getBase().equals(baseForUpgradeCommit)) {
 					logger.info("Commit {} has base {} which is different to the last upgrade commit. " +
 									"This upgrade commit will not be rolled back nor will any commits before this.",
-							commit.getHeadTimestamp(), commit.getBaseTimestamp());
-					baseForUpgradeCommit = commit.getBase();
+							lightCommit.getHeadTimestamp(), lightCommit.getBaseTimestamp());
+					baseForUpgradeCommit = lightCommit.getBase();
 					commitsToRollback.clear();
 					continue;
 				// Exclude first upgrade commit after code system version
-				} else if (baseForUpgradeCommit == null && baseForReleaseCommit != null && !commit.getBase().equals(baseForReleaseCommit)) {
+				} else if (baseForUpgradeCommit == null && baseForReleaseCommit != null && !lightCommit.getBase().equals(baseForReleaseCommit)) {
 					logger.info("Commit {} has base {} which is different to the release commit. " +
 									"This upgrade commit will not be rolled back nor will any commits before this.",
-							commit.getHeadTimestamp(), commit.getBaseTimestamp());
-					baseForUpgradeCommit = commit.getBase();
+							lightCommit.getHeadTimestamp(), lightCommit.getBaseTimestamp());
+					baseForUpgradeCommit = lightCommit.getBase();
 					commitsToRollback.clear();
 					continue;
 				} else {
 					logger.info("Commit {} does not have a new base ({}) so will be rolled back.",
-							commit.getHeadTimestamp(), commit.getBaseTimestamp());
+							lightCommit.getHeadTimestamp(), lightCommit.getBaseTimestamp());
 				}
+				Branch commit = branchService.findAtTimepointOrThrow(lightCommit.getPath(), lightCommit.getHead());
 				commitsToRollback.add(commit);
 			}
 		} else {
@@ -167,6 +168,7 @@ public class DailyBuildService {
 
 		// Roll back in reverse order (i.e the most recent first)
 		Collections.reverse(commitsToRollback);
+
 		rollbackCommits(branchPath, commitsToRollback);
 	}
 

--- a/src/test/java/org/snomed/snowstorm/dailybuild/DailyBuildServiceTest.java
+++ b/src/test/java/org/snomed/snowstorm/dailybuild/DailyBuildServiceTest.java
@@ -13,7 +13,6 @@ import org.snomed.snowstorm.TestConfig;
 import org.snomed.snowstorm.core.data.domain.CodeSystem;
 import org.snomed.snowstorm.core.data.domain.CodeSystemVersion;
 import org.snomed.snowstorm.core.data.domain.Concept;
-import org.snomed.snowstorm.core.data.services.AdminOperationsService;
 import org.snomed.snowstorm.core.data.services.CodeSystemService;
 import org.snomed.snowstorm.core.data.services.CodeSystemUpgradeService;
 import org.snomed.snowstorm.core.data.services.ConceptService;
@@ -23,17 +22,21 @@ import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.io.ResourceLoader;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
-import org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
-import java.io.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.List;
 
-import static junit.framework.TestCase.*;
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertNull;
+import static junit.framework.TestCase.assertTrue;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.*;
 
 @ExtendWith(SpringExtension.class)
 @ContextConfiguration(classes = TestConfig.class)
@@ -57,9 +60,6 @@ class DailyBuildServiceTest extends AbstractTest {
 	@Autowired
 	private CodeSystemUpgradeService codeSystemUpgradeService;
 
-	@Autowired
-	private AdminOperationsService adminOperationsService;
-
 	private File baseLineRelease;
 
 	private File rf2Archive1;
@@ -67,9 +67,6 @@ class DailyBuildServiceTest extends AbstractTest {
 	private File rf2Archive2;
 
 	private CodeSystem snomedct;
-
-	@Autowired
-	private ElasticsearchRestTemplate elasticsearchTemplate;
 
 	@Autowired
 	private ResourceLoader resourceLoader;

